Source/Logger.swift

```diff
@@ -0,0 +1,68 @@
+import Venice
+public class Logger {
+    
+    //    #namespace     String   The name of the namespace / module
+    //    #file          String   The name of the file in which it appears.
+    //    #line          Int      The line number on which it appears.
+    //    #column        Int      The column number in which it begins.
+    //    #function      String   The name of the declaration in which it appears.
+    //    #dsohandle     String   The dso handle.
+    
+    var appenders = [Appender]()
+    var levels : Log.Level
+    var name : String
+    
+    public init(name: String, appender: Appender, levels: Log.Level = .all) {
+        appenders.append(appender)
+        self.levels = levels
+        self.name = name
+    }
+    
+    public init(name: String, appenders: [Appender], levels: Log.Level = .all) {
+        self.appenders.append(contentsOf: appenders)
+        self.levels = levels
+        self.name = name
+    }
+    
+    public func log(item: Any, error: ErrorProtocol? = nil, _ file: String = #file, _ function: String = #function, _ line: Int = #line, _ column: Int = #column) {
```

Comment:
Those file, function, line, etc. arguments are a mess. I haven't tested this, but I have a feeling that something like `info: LocationInfo = LocationInfo()` where `LocationInfo` has the initializer with the million debug parameters (with default values).